GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1404,7 +1403,7 @@ void OnRevisionReadCompleted()
                         // (can occur when filtering or limiting number of loaded commits)
                         // Try to find the revision where to attach the artificial, first is not found
                         // null means that no matching should be done (just add), so use an empty list if no parents were found
-                        refresh = AddArtificialRevisions(headParents ?? Array.Empty<ObjectId>());
+                        _ = AddArtificialRevisions(headParents ?? []);
```

Comment:
side note: This handling would have been easier if aritificial was always added at the top.
But I prefer them at the HEAD as they are now.